e2e/cypress/integration/channel/convert_channel_to_private_spec.js

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @channel
+
+describe('Channels', () => {
+    let testUser;
+    let testTeam;
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user}) => {
+            testUser = user;
+            testTeam = team;
+        });
+    });
+
+    it('MM-28274 - Convert to private channel should only be shown to users with permission', () => {
```

Comment:
Can you please rename the test id from `MM-28274` to `MM-T3348`, since now we have to map to the TM4J test ID instead of JIRA ticket ID.

Reply:
This is done now, just waiting on @michaelgamble to review then I can merge...